gulp-test-chrome-debug.js

```diff
@@ -0,0 +1,75 @@
+const { series } = require('gulp');
+const del = require('del');
+const run = require('gulp-run-command').default;
+
+const IS_WINDOWS = (process.platform == 'win32');
+
+/**
+ * Run a command in the vscode-chrome-debug working directory
+ */
+function runInDebug(cmd) {
+    return run(cmd, { cwd: './vscode-chrome-debug' })();
+}
+
+/**
+ * Delete the vscode-chrome-debug working directory (if it exists)
+ */
+function clean(done) {
+    // using sync because async del doesn't report failures correctly, can can cause hangs
+    del.sync(['vscode-chrome-debug']);
+    done();
+}
+
+/**
+ * Clone the vscode-chrome-debug project from GitHub
+ */
+async function clone() {
+    console.log('git is located at:');
+    await run('where git')();
+    if(IS_WINDOWS)
+        return run('git clone -b v2 --single-branch --depth 1 http://github.com/Microsoft/vscode-chrome-debug.git', { env: process.env })();
+    else
+        return run('git clone -b v2 --single-branch --depth 1 http://github.com/Microsoft/vscode-chrome-debug.git')();
+}
+
+/**
+ * Run `npm install` on vscode-chrome-debug
+ */
+function install() {
+    return runInDebug('npm install');
+}
+
+/**
+ * Substitute our version of -core for the tests
+ */
+async function setCoreVersion(cb) {
```

Comment:
I'd rename this to setCoreVersion linkChromeDebugToCore
setCoreVersion makes me think that you are setting the version as in 4.5.6